samples/dnn/js_face_recognition.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+  <script src="opencv.js" type="text/javascript"></script>
+  <script src="utils.js" type="text/javascript"></script>
```

Comment:
In my test, browser reports 404 not found error when loading `opencv.js` and `utils.js`. The reason is this page is copied to a folder (e.g. `d5/d86/tutorial_dnn_javascript.html`). However, `opencv.js` and `utils.js` is copied to doc root.

Reply:
Thanks! Replaced to `../../opencv.js` and `../../utils.js`.